### PR TITLE
Use OSSF scorecard

### PR DESCRIPTION
**Pull request by anderseknert**

### CODE REVIEW

File: .github/workflows/scorecard.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Scorecard supply-chain security
+on:
+  branch_protection_rule:
+  schedule:
+    - cron: '16 9 * * 2'
```
Comment (srenatus) [inline .github/workflows/scorecard.yml:5]: Now that's a funny-looking cron. crontab.guru says
> “At 09:16 on Tuesday.” 

OK fine.

File: .github/workflows/scorecard.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    runs-on: ubuntu-latest
+    permissions:
+      # Needed to upload the results to code-scanning dashboard.
+      security-events: write
+      # Needed to publish results and get a badge (see publish_results below).
+      id-token: write
+
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```
Comment (srenatus) [inline .github/workflows/scorecard.yml:23]: I like this. We should do this more often.

**Review by srenatus — APPROVED** [review]
LGTM

File: .github/workflows/scorecard.yml (hunk 1 above)
Comment (anderseknert) [reply .github/workflows/scorecard.yml:5]: Oddly specific 😆

File: .github/workflows/scorecard.yml (hunk 2 above)
Comment (anderseknert) [reply .github/workflows/scorecard.yml:23]: Indeed... something tells me that's one of the things this tool is going to tell us 😅

**Pull request merged**